tables_en/2-02.csv

```diff
@@ -332,31 +317,26 @@ OSWHOI,\Co-sponsored\GOOS\OceanSITES\OS-WHOI,OS-WHOI,NA
 OSWHOI-SIO-OSU,\Co-sponsored\GOOS\OceanSITES\OS-WHOI/SIO/OSU,OS-WHOI/SIO/OSU,NA
 pacIOOS,\Co-sponsored\GOOS\DBCP\PACIOOS,PACIOOS,Pacific Islands Ocean Observing System Stations
 petrobras,\Co-sponsored\GOOS\DBCP\PETROBRAS,PETROBRAS,NA
-PHOTONS,\WIGOS\GAW\GAW Other elements\AOD\PHOTONS,PHOTONS,"PHOTONS, federated to AERONET"
+PHOTONS,\WIGOS\GAW\GAW Other elements\PHOTONS,PHOTONS,"PHOTONS, federated to AERONET"
 PMELMB,\Co-sponsored\GOOS\DBCP\PMEL MB,PMEL MB,PMEL Moored Buoys
-polarAOD,\WIGOS\GAW\GAW Other elements\AOD\PolarAOD,PolarAOD,Polar Aerosol Optical Depth Measurement Network Project
-RBCN,\WIGOS\GOS\GOS Surface networks\RBCN,RBCN,Regional Basic Climatological Networks
-RBONpilot,\WIGOS\GOS\GOS Surface networks\RBON-Pilot Project,RBON-Pilot Project,Regional Basic Observing Network (pilot project)
-RBSN,\WIGOS\GOS\GOS Surface networks\RBSN,RBSN,GOS Regional Basic Synoptic Networks
-RBSNp,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(P),RBSN(P),Regional Basic Synoptic Network upper-air station (PILOT)
-RBSNs,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(S),RBSN(S),Regional Basic Synoptic Network surface station (SYNOP)
-RBSNsp,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(SP),RBSN(SP),NA
-RBSNst,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(ST),RBSN(ST),Regional Basic Synoptic Network surface and upper-air station (SYNOP/TEMP)
-RBSNt,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(T),RBSN(T),Regional Basic Synoptic Network upper-air station (TEMP)

```

Comment:
@amilan17 So, all these RBCN and RBSN entries will be listed as superseded by RBON, correct? Are we supposed to re-assign all existing affiliations in OSCAR/Surface to RBON as well, or will we keep them? No immediate response required, but we need to agree how OSCAR/Surface should foow-up such changes.

Reply:
@joergklausen  - Yes, these terms are superseded by RBON and this will be apparent in the Codes Registry.

Reply:
@joergklausen - please resolve this conversation if you are satisfied with my answer.